### ISSUE TITLE
PiDP-10 and TOPS-10

### ISSUE THREAD

**Issue by statespacedev** [drforbin/decwar#4]
Hi, with pidp-10 tops-10 suddenly going here - very first question is about getting decwar running on it:) Is anyone out there already successfully doing so? Anyone interested in discussion here?:)

Cheers,
Noah

**Comment by drforbin** [comment]
It should work just fine.
Let me know if I can help.

This is drforbin. the original guy who got it running on tops10

**Comment by statespacedev** [comment]
Thanks drforbin - that's great news!:) Just giving a quick reply at the moment - going to get thoughts together over this weekend and will come back with more right here in the next few days!
more soon!

**Comment by larsbrinkhoff** [comment]
Maybe another challenge in the future, Empire: https://github.com/DigitalMars/Empire-for-PDP-10/issues/1

**Comment by statespacedev** [comment]
While working on my notes for this, got excited by rediscovering this - remember seeing it sometime ago, memory was hazy - just have to give a quick yell right away!:)

_Merlyn Cousins (Drforbin) later reverse engineered the source back to a form which would run on standard TOPS-10 and simh. This code is archived at the Briscoe Center for American History.[8] The source is also available on GitHub.[9])_ [link](https://en.wikipedia.org/wiki/Decwar#Other_versions)

here's draft post am working on - https://statespace.dev/tops-10.html 

am thinking the thing to do is simply ask questions, try to figure stuff out, and do something in return by trying to document what happens - lots to learn, can try to share along the way:)

**Comment by drforbin** [comment]
Give me your email...I'll give you my number via email and we can talk

I have a tops10 system up and running. I can make you an account to play around with decwar
The original MUD etc if you wish.

**Comment by statespacedev** [comment]
Thanks drforbin - sounds good, would like that! Would be great to get 'intro advice' on good ways to get into into all this:) You can email me at 'noah at statespace dot dev'

Would indeed like to get an account and see it in action - mostly out of curiosity to learn about how it all works now - was never a champ decwar player back in the day 83/84, more fascinated by how it worked and how it brought people together somehow. Would like to help preserve that and make it more accessible and visible, as it were:)

**Comment by statespacedev** [comment]
Hi drforbin - quick follow-up and a fortran question:) Have slowly been getting up to speed on tops-10 - am now doing basic stuff in macro and fortran and basic things are working ok. Feeling ready to at least start looking at the decwar code:) Looking like very roughly ~75 .for files and ~10 .mac files - so definitely going to be learning lots about for and mac:)

Quick question - have docs on both fortran IV and fortran-10 - so f40 and f10 as seems they sometimes say. Are there known dependencies here for decwar - either
f40 (left) or f10 (right)?
![PXL_20240816_230810775](https://github.com/user-attachments/assets/cf3def3a-b973-4693-8ae7-d9fdde5897db)

**Comment by drforbin** [comment]
I thought you were going to give me a call?
Anyway You need an old version of FORTAN for decwar to compile on.
We can do this in various ways. I can provide you have a rdy made copy of tops-10 which has everything installed or I can find the tap for you to install or I can just tell you the version.
Besides that, what are you asking?

**Comment by statespacedev** [comment]
Oh dear! Looks like I flubbed up, I remembered  my email so we could set up a call, but didn't see an incoming email - definitely want to chat!!! - here's my email in normal fashion noah@statespace.dev (not really sure why have been seeing the 'noah at statespace dot dev' type thing, but did it anyway up above:(

Will give you a call soon as possible! Sounds like having the copy of tops-10 with everything will be best for learning... will go whatever direction you thinks best, for sure! 

Besides that, if possible am eventually asking lots of questions probly:) From my side, could contribute documentation - 'newbie learns how to work with/on decwar':) and history, etc - looking like will have the time to put some solid work in here, could be on 'documentation type stuff'?:)
